crates/bitwarden-exporters/src/lib.rs

```diff
@@ -132,3 +134,12 @@ pub struct Identity {
     pub passport_number: Option<String>,
     pub license_number: Option<String>,
 }
+
+pub struct SshKey {
+    /// Ssh private key (ed25519/rsa) in unencrypted openssh private key format https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.key
```

Comment:
From the tests I believe it's actually the "PEM" encoding of the OpenSSH private key format.

```suggestion
    /// [OpenSSH private key](https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.key), in PEM encoding.
```